src/store.js

```diff
@@ -0,0 +1,18 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+Vue.use(Vuex);
+
+export default new Vuex.Store({
+  state: {
+    previewUserId: null,
+  },
+  mutations: {
+    updatePreviewProfileId(state, update) {
+      state.previewUserId = update.newId; // eslint-disable-line no-param-reassign
```

Comment:
@fiji-flo our eslint does not like me changing something that's passed as param. Vuex docs [recommend](https://vuex.vuejs.org/guide/mutations.html#mutations) it. What are your thoughts?